pkg/proxy/util/conntrack.go

```diff
@@ -31,17 +31,21 @@ const noConnectionToDelete = "0 flow entries have been deleted"
 
 // DeleteServiceConnection uses the conntrack tool to delete the conntrack entries
 // for the UDP connections specified by the given service IPs
-func DeleteServiceConnections(execer exec.Interface, svcIPs []string) {
+func DeleteServiceConnections(execer exec.Interface, svcIPs []string) error {
+	var errs []error
 	for _, ip := range svcIPs {
```

Comment:
I Fixed it by refactoring `DeleteServiceConnections()` only handle one service IP at a time. PTAL.